proto/backup.proto

```diff
@@ -39,17 +39,25 @@ message BackupMeta {
 
 message File {
     string name = 1;
-    uint32 crc32 = 2;
+    uint64 content_checksum = 2;
```

Comment:
Could you give it a self-described name?

Reply:
like the used algorithm name?